src/pages/assetGroup/AssetGroup.jsx

```diff
@@ -146,32 +150,33 @@ export default function AssetGroup() {
           </Text>
         )}
       </EntityHeader>
-      {showPreparationErrorAlert && (
+      {isPreparationFailed && (
         <CustomAlert
           titleId="IMAGE_PROCESSING_ERROR"
           descriptionId="IMAGE_PROCESSING_ERROR_MESSAGE"
           severity="error"
         />
       )}
-      {showPreparationInProgressAlert && (
-        <>
-          <LinearProgress
-            style={{
-              borderTopLeftRadius: '4px',
-              borderTopRightRadius: '4px',
-            }}
-          />
-          <CustomAlert
-            titleId="PENDING_IMAGE_PROCESSING"
-            descriptionId="PENDING_IMAGE_PROCESSING_MESSAGE"
-            severity="info"
-            style={{
-              borderTopLeftRadius: '0px',
-              borderTopRightRadius: '0px',
-            }}
-          />
-        </>
-      )}
+      {!isPreparationFailed &&
+        !isProgressSettled(pipelineStatusPreparation) && (
```

Comment:
i don't love these nested `&&`s. possible to set a variable to `isPreparationFailed && !isProgressSettled(pipelineStatusPreparation)` above?

Reply:
Yes. I restored the `showPreparationInProgressAlert` variable in https://github.com/WildMeOrg/codex-frontend/pull/420/commits/ec3b7a0e0ff315dd51dbf03e6f9b56102c07a027